src/containers/FormikForm/formikSubjectpageHooks.ts

```diff
@@ -35,33 +35,35 @@ export function useFetchSubjectpageData(
   const [error, setError] = useState(undefined);
 
   const fetchElementList = async (taxonomyUrns: string[]) => {
-    const taxonomyElements = await Promise.all(
+    const taxonomyElements = await Promise.all<{ id: string; contentUri?: string }>(
```

Comment:
Blir ikke typen `Topic | Resource` her?

Reply:
For subjectpage kan det vel også være Subject?